clients/js/src/packData.ts

```diff
@@ -133,27 +140,47 @@ export function unpackExternalData(data: ReadonlyUint8Array): {
   };
 }
 
-export async function unpackAndFetchExternalData(
-  input: Omit<PackedData, 'dataSource'> & { rpc: Rpc<GetAccountInfoApi> }
-): Promise<string> {
-  const externalData = unpackExternalData(input.data);
-  const account = await fetchEncodedAccount(input.rpc, externalData.address);
-  assertAccountExists(account);
+export function uncompressAndDecodeExternalData({
```

Comment:
Since the convention of this file is that "unpack" means decompressing (if needed) and decoding, how about naming this function `unpackFetchedExternalData`? That way we communicate that we are doing the same as these other unpack methods but for the fetched content of an external data source.